src/core/api/clock.ts

```diff
@@ -337,11 +337,12 @@ function createClock(
     function getCurrentClockTick(state : IMediaInfosState) : IClockTick {
       const mediaTimings = getMediaInfos(mediaElement, state);
       const stalledState = getStalledStatus(lastTimings, mediaTimings, options);
+      log.debug("API: current media element state", objectAssign({}, { stalled: stalledState }, mediaTimings));
```

Comment:
Why are we doing the job of objectAssign two times?
Why not just constructing the returned tick before logging